### ISSUE TITLE
VTK not registering with conda: preventing other vtk-dependent installs

### ISSUE THREAD

**Issue by banesullivan** [conda-forge/paraview-feedstock#83]
When installing ParaView via this recipe, it seems to also build VTK and that VTK is available in the conda environment's site-packages under `vtkmodules`, however, it does not register with the conda package manager. 

This means conda will not pick up on that installation of VTK - currently, I cannot install any other package that relies on VTK as the environment isn't compatible because of this.

A hacky workaround is to install VTK (and other VTK-dependent packages) from pip into the conda env (*cringe*) and then things work, but... if you then try to import VTK in the `pvpython` env, a segfault occurs as that version of VTK isn't properly linked to ParaView (or something... this is getting out of my knowledge).

For example, run my docker image from https://hub.docker.com/repository/docker/banesullivan/paraview which shows the Dockerfile online (how I installed ParaView from this recipe). When you do a `conda list`, `vtk` does not appear, however in Python, `import vtk` succeeds with it pointing to `/opt/conda/lib/python3.7/site-packages/vtkmodules/all.py`

So. **How do we properly set up VTK in the environment so that other packages that rely on VTK like [PyVista](https://github.com/conda-forge/pyvista-feedstock) can also be installed in the environment with conda?**

**Comment by jschueller** [comment]
we cant, it's a known issue that vtk and paraview installations clash
linux distributions also suffer from this: eg https://bugs.archlinux.org/task/61596
i think the root of all evil is that paraview's internal vtk is not in sync with vtk releases
please open an issue on https://gitlab.kitware.com/paraview/paraview

**Comment by banesullivan** [comment]
Is there a way we could at least trick `conda` into thinking the `vtkmodules` installed with ParaView is a valid package so that it sees that when building other recipes?

**Comment by jschueller** [comment]
maybe with a subpackage, but its version would not be clearly defined as it's not from a vtk release.

**Comment by banesullivan** [comment]
I got this to work. Check out what I did here: https://github.com/banesullivan/dockerfiles/blob/master/paraview/Dockerfile#L24-L29

It's a total hack, but it works reliably for me. It just moves the `vtk.py` file created by ParaView into a new package and installs it via `pip` so if you try to install any packages that rely on VTK, they will see that version of VTK.

This likely won't work too well with conda, but conda shows this VTK package now when doing a `conda list`

Also, the Docker image is published here: https://hub.docker.com/repository/docker/banesullivan/paraview
